frontend/app/routes/$lang+/_public+/apply+/$id+/applicant-information.tsx

```diff
@@ -157,9 +158,9 @@ export default function ApplyFlowApplicationInformation() {
             <FontAwesomeIcon icon={faChevronLeft} className="me-3 block size-4" />
             {t('applicant-information.back-btn')}
           </ButtonLink>
-          <Button variant="primary" id="continue-button">
+          <Button variant="primary" id="continue-button" disabled={navigation.state === 'submitting'}>
             {t('applicant-information.continue-btn')}
-            <FontAwesomeIcon icon={faChevronRight} className="ms-3 block size-4" />
+            <FontAwesomeIcon icon={navigation.state === 'submitting' ? faSpinner : faChevronRight} className={`ms-3 block size-4 ${navigation.state === 'submitting' ? 'animate-spin' : ''}`} />
```

Comment:
For this I think you should use

```
className={cn('ms-3 block size-4', navigation.state !== 'idle' && 'animate-spin')}
```

since the `cn()` function was designed for this specific purpose